### DISCUSSION TITLE
Call code client side only

### DISCUSSION THREAD

**Discussion by remy** [nuxt/nuxt#7878] [General]
_This isn't a question about `<client-only>`_

My context is this: I have a tabbing system that when rendered on the server the tabs are unrolled - as in all content is visible (and viewable by SEO crawlers).

Once the page has loaded in the browser, I want to run code that will progressively enhance the tabs into panels that show and hide. A fairly standard tab dev pattern.

I can't work out how to achieve this at all with Nuxt. It seems that if my code that turns the markup into tabs works conditionally with `process.client`, the server renders the tabs unrolled and the client side mount isn't called (nor the initial `data()` method).

Any suggestions? (and please don't suggest `<client-only>` unless you can point me to a specific example does indeed progressively enhance tabs).

Thank you in advance.

**Comment by danielroe** [comment]
@remy You could try a plugin. I run conditional enhancement on links and images this way. Name it `~/plugins/somename.client.js` and include it in your `nuxt.config.js` plugins array and Nuxt will only load it on the client-side. [More info here](https://nuxtjs.org/guide/plugins/#client-or-server-side-only).
